tpm/tpm.go

```diff
@@ -975,6 +975,40 @@ func GetManufacturer(rw io.ReadWriter) ([]byte, error) {
 	return getCapability(rw, capProperty, tpmCapPropManufacturer)
 }
 
+// GetAlgs returns a slice of bytes.
+// A byte is 1 a supported algorithm
+func GetAlgs(rw io.ReadWriter) ([]byte, error) {
+	var algs []byte
+	for i := uint32(1); i <= 10; i++ {
+		buf, err := getCapability(rw, capAlg, i)
+		if err != nil {
+			return nil, err
+		}
+		if buf == nil {
+			return nil, fmt.Errorf("Function GetAlgs() returned no error and no value")
+		}
+		for _, item := range buf {
+			algs = append(algs, item)
```

Comment:
I don't know how to read the 1.2 spec yet.
What do you expect to see in buf for each getCapability request?

Reply:
The spec doesn't tell me how many bits/bytes are returned to repesent the boolean. I just testes it and i just get one byte.